new-pipeline/src/evo.js

```diff
@@ -26,7 +26,7 @@ var gDefaultAggregates = [
   }],
 ];
 // these will be generated, but won't appear in the multiselect
-// note: some code using gMetaAggregates assumes for convenience that 
+// note: some code using gMetaAggregates assumes for convenience that
```

Comment:
There are whitespace changes in this file that we don't need.
Please drop these changes.

Reply:
Yep. Actually all these changes are occurring because my editor has a different default indentation. And so if I open a file by mistake, it gets altered.

I'll checkout these changes